time-elements.js

```diff
@@ -561,7 +563,9 @@
     var options = {
       hour: el.getAttribute('hour'),
       minute: el.getAttribute('minute'),
-      second: el.getAttribute('second')
+      second: el.getAttribute('second'),
+      timeZone: el.getAttribute('timeZone'),
+      timeZoneName: el.getAttribute('timeZoneName')
```

Comment:
@mislav, the purpose for setting e.g. `timezone=PST` is to explicitly be able to force the timezone used. It defaults to UTC.

> [The time zone to use. The only value implementations must recognize is "UTC"; the default is the runtime's default time zone. Implementations may also recognize the time zone names of the IANA time zone database, such as "Asia/Shanghai", "Asia/Kolkata", "America/New_York".
](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat)

I agree that it is not necessary, but if adding `timeZoneName` I thought to also add `timeZone` for [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat).